src/app/calculator/waste-water/state-point-analysis/state-point-analysis-form.service.ts

```diff
@@ -34,6 +36,8 @@ export class StatePointAnalysisFormService {
       sviParameter = updatedBaselineInput.sviParameter;
       numberOfClarifiers = updatedBaselineInput.numberOfClarifiers;
       areaOfClarifier = updatedBaselineInput.areaOfClarifier;
+      diameter = updatedBaselineInput.diameter;
+      userDefinedArea = updatedBaselineInput.userDefinedArea;
```

Comment:
Change name throughout to **is**UserDefinedArea so we immediately know its a boolean.